tests/console-backend-service/internal/configurer/broker.go

```diff
@@ -0,0 +1,142 @@
+package configurer
+
+import (
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
+	"github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset"
+	tester "github.com/kyma-project/kyma/tests/console-backend-service"
+	"github.com/kyma-project/kyma/tests/console-backend-service/internal/waiter"
+)
+
+type ServiceBrokerConfig struct {
+	Name      string `envconfig:"default=ns-helm-broker"`
```

Comment:
Maybe we should add information that it is for CBS tests

Reply:
Good idea - I will add test label for all resources I create.